forecasts.js

```diff
@@ -306,58 +296,50 @@ forecasts.getRegionDetailsForRegionId = function(regionId) {
             var parser = null;
             switch (components[0]) {
                 case 'nwac':
-                    dataURL = 'http://www.nwac.us/api/v2/avalanche-region-forecast/?format=json&limit=1&zone=' + components[1];
+                    dataURL = 'https://www.nwac.us/api/v2/avalanche-region-forecast/?format=json&limit=1&zone=' + components[1];
                     parser = forecasts.parseForecast_nwac;
                     break;
                 case 'cac':
                     // CAC South Coast has two polygons that have the same forecast; handle that
                     var subRegion = components[1].split('_')[0];
-                    dataURL = 'http://www.avalanche.ca/api/forecasts/' + subRegion + '.json';
+                    dataURL = 'https://www.avalanche.ca/api/forecasts/' + subRegion + '.json';
                     parser = forecasts.parseForecast_cac;
                     break;
                 case 'cacb': // CAC blog-only forecasts, which we don't parse
-                    dataURL = 'http://www.avalanche.ca/blogs?category=' + components[1];
+                    dataURL = 'https://www.avalanche.ca/blogs?category=' + components[1];
```

Comment:
thanks for moving all these to https. 

have you verified that all these URLs resolve correctly?  thanks

Reply:
I verified all of them. The only center not using https right now is COAA
Of course, it would not hurt to have another person check them too :)

Reply:
great!